examples/index.md

```diff
@@ -162,3 +162,354 @@ Content-Type: application/vnd.api+json
+      "detail": "First name must contain an emoji."
+    }
+  ]
+}
+```
+
+A "400 Bad Request" response would also be acceptable. See
+[http://stackoverflow.com/a/20215807/1261879]('400 vs 422 response to POST of data' on
+Stack Overflow). The JSON API doesn't take a position on 400 vs. 422.
+
+If the API docs specified:
+
+> | Error Code | Title                                                                             |
+> |------------|-----------------------------------------------------------------------------------|
+> |  11        | "Invalid Attribute"                                                               |
+> |  123       | "too short"                                                                       |
+> |  124       | "emoji missing"                                                                   |
+> |  225       | "password must contain a letter, number, space, neologism, and special character" |
+> |  226       | "passwords do not match"                                                          |
+> |  227       | "password cannot be one of last five passwords"                                   |
+
+Multiple errors on `password` attribute, with error "code":
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "code":   "123",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain at least three characters."
+    },
+    {
+      "code":   "225",
+      "source": { "pointer": "data/attributes/password" },
+      "detail": "Frobnicate is not a neologism."
+    },
+    {
+      "code":   "226",
+      "source": { "pointer": "data/attributes/password" },
+      "detail": "Password and password confirmation do not match."
+    }
+  ]
+}
+```
+
+Multiple errors on `first-name` attribute, with error "code", and "title":
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "jsonapi": { "version": "1.0" },
+  "errors": [
+    {
+      "code":   "123",
+      "title":  "too short",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain at least three characters."
+    },
+    {
+      "code":   "124",
+      "title":  "emoji missing",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain an emoji"
+    }
+  ]
+}
+```
+
+Notice that this response includes not only the "errors" top-level member,
+but the "jsonapi" top-level member. It may also have include the "meta" or "links"
+top-level members, but not the "data" member.
+
+```http
+PATCH /posts/1
+Content-Type: application/vnd.api+json
+Accept: application/vnd.api+json
+
+{ "datum": [ ] }
+```
+
+```http
+HTTP/1.1 422 Unprocesssable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "title":  "Missing Data Member",
+      "source": { "source": "/data" }
```

Comment:
Is the 2nd `source` a mistake? Spec only mentions `pointer` and `parameter`. There's another occurrence of this in the very last example if so.